.travis.yml

```diff
@@ -16,6 +19,9 @@ jobs:
   allow_failures:
     - go: master
   include:
+    - stage: deploy-docker
```

Comment:
I'll run in parallel but I think it's more readable to put this stage after `test` and `format`, wdyt?